recipe/yum_requirements.txt

```diff
@@ -0,0 +1,9 @@
+libX11
+libXcomposite
+libXcursor
+libXdamage
+libXext
+libXfixes
+libXi
+libXinerama
+libXrandr
```

Comment:
Why was this file added?

Reply:
I added `pygobject` as a downstream test, since it is the only recipe that has a run-time dependency on `gobject-introspection`, and problems with this recipe are most likely to show up there.

Since the `pygobject` tests depend on `gtk3`, I need to pull in these yum dependencies.